### ISSUE TITLE
# OF-Scraper Announcements:__(1) error, cant get past message.

### ISSUE THREAD

**Issue by god43** [datawhores/OF-Scraper#492]
hello I cant seem to run the scrapper, I cant get past this message and I am not sure why? I set it to keydb and provided an api key. thanks.

OF-Scraper Announcements:__(1). For those who use cdrm for cdm; a fix has    logs.py:82
been pushed to handle errors from the CDRM-Project API. Previously FFMPEG would endlessly fail for effected
downloads because of bad inputs being cached. Update to at least [3.12.7]__(2). cdrm seems to be getting ip
blocked by onlyfans based on error messages, the fix is to use your own keys or switch to keydb or to wait
for the requests to work again

**Comment by cjb900** [comment]
This isn't an error message but just an announcement message I the scraper. To help we would need a screenshot of the issue you are having to try and help.

**Comment by god43** [comment]
It's just that message, a screenshot wouldn't show you much different. I launch the scraper with of scraper command, this message appears but then automatically the script finishes before providing me any options to choose.
________________________________

**Comment by cjb900** [comment]
If a screenshot wouldn't be helpful then I wouldn't have asked for one.

**Comment by god43** [comment]
here you go, thanks.

![image](https://github.com/user-attachments/assets/f2169450-fa79-440a-95f7-1092caded2a2)

**Comment by cjb900** [comment]
Ya this shows more information than what you initially posted and tells me exactly what your issue is. You need to have `aiolimiter` version 1.1.0 installed. Any newer version will not work.

**Comment by god43** [comment]
interesting ok thanks ill try that out and let you know.

**Comment by ScottP8** [comment]
> Ya this shows more information than what you initially posted and tells me exactly what your issue is. You need to have `aiolimiter` version 1.1.0 installed. Any newer version will not work.

Hi, thanks for your hint, I got the same error message, then I installed “aiolimiter” 1.1.0, it but it seems still not working, could you tell me if I miss anything please?
<img width="851" alt="ofscraper issue 492" src="https://github.com/user-attachments/assets/e8480b9b-653f-4c84-a4a5-d9d20d86c4e1" />

**Comment by cjb900** [comment]
If you have confirmed to have `aiolimiter` version 1.1.0 installed you may need to reinstall ofscraper to get it to work.

**Comment by ScottP8** [comment]
> If you have confirmed to have `aiolimiter` version 1.1.0 installed you may need to reinstall ofscraper to get it to work.

Yes, ok just tried, still got the identical message of the screenshot above
![ofscraper issue 492 p3](https://github.com/user-attachments/assets/7c5ccfef-38b9-4e92-b521-28f109c210f7)

**Comment by cjb900** [comment]
you need to uninstall the pipx version of ofscraper and install it via pip to get it to work with the way you installed `aiolimiter`

**Comment by god43** [comment]
i installed it using "pip install aiolimiter==1.1.0", and my issue went away. thanks.

**Comment by ScottP8** [comment]
> you need to uninstall the pipx version of ofscraper and install it via pip to get it to work with the way you installed `aiolimiter`

Now seems working to me too, thanks a lot

**Comment by WyohKnott** [comment]
@datawhores can you pin an issue regarding this error?

**Symptom:** ofscraper closes with no download with "Finished script"

**Solution:** Pin aiolimiter to version 1.1.0:

 - if you used pip to install: `pip install aiolimiter==1.1.0`
 - if you used pipx to install: `pipx inject ofscraper aiolimiter==1.1.0`
